src/contentApi/get-apps-data.js

```diff
@@ -7,6 +7,8 @@ import { createRhelSchema } from './rhel';
 
 import { getManagedServicesDataSchema } from './managed-services-api';
 
+const env = window.insights.chrome.getEnvironment();
```

Comment:
Isn't there a way how we could pass this from hook? I don't like `window.insights` approach anymore and I'd rather avoid adding more tech debt.